### PR TITLE
Wifi util shell

### PR DESCRIPTION
**Pull request by ajayparida**

### CODE REVIEW

File: drivers/wifi/nrf700x/zephyr/src/zephyr_wifi_util.h [hunk 1]
```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright (c) 2022 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+/* @file
+ * @brief nRF Wi-Fi radio-test mode shell module
```
Comment (sachinthegreen) [inline drivers/wifi/nrf700x/zephyr/src/zephyr_wifi_util.h:8]: `radio-test` ?

File: samples/wifi/shell/prj.conf [hunk 2]
```diff
@@ -5,6 +5,7 @@
 #
 CONFIG_WIFI=y
 CONFIG_WIFI_NRF700X=y
+CONFIG_NRF700X_WIFI_UTIL=y
```
Comment (sachinthegreen) [inline samples/wifi/shell/prj.conf:8]: We should not have this enabled by default.

**Review by sachinthegreen — CHANGES_REQUESTED** [review]

File: drivers/wifi/nrf700x/zephyr/src/zephyr_wifi_util.c [hunk 3]
```diff
@@ -0,0 +1,307 @@
...
+}
+
+
+static int nrf_wifi_util_set_he_ltf(const struct shell *shell,
+					  size_t argc,
+					  const char *argv[])
+{
+	char *ptr = NULL;
+	unsigned long he_ltf = 0;
+
```
[the remark this answers is not in the document]
Comment (ajayparida) [reply drivers/wifi/nrf700x/zephyr/src/zephyr_wifi_util.c:42]: No, it is alrady part of command (minimum arguments required)

File: drivers/wifi/nrf700x/zephyr/src/zephyr_wifi_util.c [hunk 4]
```diff
@@ -0,0 +1,307 @@
...
+
+static int nrf_wifi_util_set_he_ltf_gi(const struct shell *shell,
+					 size_t argc,
+					 const char *argv[])
+{
+	enum wifi_nrf_status status = WIFI_NRF_STATUS_FAIL;
+	char *ptr = NULL;
+	unsigned long val = 0;
+
```
[the remark this answers is not in the document]
Comment (ajayparida) [reply drivers/wifi/nrf700x/zephyr/src/zephyr_wifi_util.c:91]: No it is already part of the command (minimum arguments required)

File: drivers/wifi/nrf700x/zephyr/src/zephyr_wifi_util.h (hunk 1 above)
Comment (sachinthegreen) [reply drivers/wifi/nrf700x/zephyr/src/zephyr_wifi_util.h:8]: Still shows as Wi-Fi radio-test mode ?

**Review by sachinthegreen — APPROVED** [review]

**Review by rlubos — APPROVED** [review]

**Pull request merged**